.circleci/bin/test-airflow-image.py

```diff
@@ -106,6 +106,79 @@ def test_astronomer_airflow_check_version(webserver):
         "astronomer-airflow-version-check module must be greater than 1.0.0"
 
 
+def test_airflow_connections(scheduler):
+    """Test Connections can be added and deleted"""
+    test_conn_uri = "postgresql://postgres_user:postgres_test@1.1.1.1:5432"
+    test_conn_id = "test"
+
+    # Assert Connection can be added
+    assert f"Successfully added `conn_id`={test_conn_id} : {test_conn_uri}" in scheduler.check_output(
+        f'airflow connections -a --conn_uri "{test_conn_uri}" --conn_id {test_conn_id}')
```

Comment:
Can you do this via array rather than string

```suggestion
    assert f"Successfully added `conn_id`={test_conn_id} : {test_conn_uri}" in scheduler.check_output(
        ['airflow', 'connections', '-a', '--conn_uri', test_conn_uri, '--conn_id', test_conn_id'])
```

Reply:
The `check_output`command takes string. Note this is different that the subprocess`.check_output`

Reply:
https://github.com/philpep/testinfra/blob/8d4a1aa65ff1cd7c45c3b16e1d11a1abc9c79d5a/testinfra/host.py#L96

Reply:
Ah, but it does have this: `host.run("ls -l -- %s", "/;echo inject")`, so:

```suggestion
    assert f"Successfully added `conn_id`={test_conn_id} : {test_conn_uri}" in scheduler.check_output(
        'airflow connections -a --conn_uri %s --conn_id %s', test_conn_uri, test_conn_id)
```

Reply:
Updated